sources/packages/web/src/views/aest/institution/Programs.vue

```diff
@@ -1,13 +1,215 @@
 <template>
   <content-group>
-    <p class="category-header-large color-blue">Programs</p>
+    <p class="category-header-large color-blue">
+      All Programs ({{ institutionProgramsSummary.programsCount }})
+      <v-row class="float-right">
+        <v-col>
+          <InputText
+            name="searchProgramName"
+            v-model="searchProgramName"
+            placeholder="Search Program Name"
+          />
+        </v-col>
+        <v-col>
+          <v-btn
+            color="primary"
+            @click="goToSearchProgramName(searchProgramName)"
+            >Search</v-btn
+          >
+        </v-col>
+        <v-col>
+          <v-btn color="primary" @click="goToSearchProgramName('')"
+            >Clear</v-btn
+          >
+        </v-col>
+      </v-row>
+    </p>
+    <content-group v-if="programsFound">
+      <DataTable
+        :value="institutionProgramsSummary.programsSummary"
+        :lazy="true"
+        :paginator="true"
+        :rows="DEFAULT_ROW_SIZE"
+        :rowsPerPageOptions="[10, 20, 50]"
```

Comment:
this list also we can pass as variable, may be we can make it as a default value